salt/beacons/__init__.py

```diff
@@ -205,10 +205,10 @@ def list_beacons(self):
         '''
         # Fire the complete event back along with the list of beacons
         evt = salt.utils.event.get_event('minion', opts=self.opts)
-        b_conf = self.functions['config.merge']('beacons')
-        if not isinstance(self.opts['beacons'], dict):
-            self.opts['beacons'] = {}
-        self.opts['beacons'].update(b_conf)
+        #b_conf = self.functions['config.merge']('beacons')
```

Comment:
Please remove these comments. We don't like to have dead code in the codebase. Thanks.